.github/workflows/composer-diff.yml

```diff
@@ -0,0 +1,34 @@
+# See https://github.com/marketplace/actions/composer-diff
+name: Composer Diff
+on:
+  pull_request:
+    paths:
+      - 'composer.lock'
+jobs:
+  composer-diff:
+    name: Composer Diff
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Required to make it possible to compare with PR base branch
+
+      - name: Generate composer diff
+        id: composer_diff # To reference the output in comment
+        # Use master branch until next release.
+        uses: IonBazan/composer-diff-action@master
```

Comment:
🤔 should this be pinned to a version?

Reply:
I think we should proceed as is. master has fixes to code and docs that we are using here.